tests/README.md

```diff
@@ -5,3 +5,7 @@ cd ./tests/
 wget https://phar.phpunit.de/phpunit.phar
 php phpunit.phar --bootstrap bootstrap.php
 ```
+
+The `shellchecks.sh` script is used to safeguard shell scripts from common
+shell script bugs and to ensure a consistent style.
+It requires [ShellCheck](https://www.shellcheck.net/) and [shfmt](https://github.com/mvdan/sh).
```

Comment:
Is `shfmt` available from `apt` or something?

Reply:
Only way I know. Could probably be adopted to always grab the latest release.
``wget https://github.com/mvdan/sh/releases/download/v2.6.4/shfmt_v2.6.4_linux_amd64 -O $HOME/.local/bin/shfmt``